packages/components/src/InputFile/InputFile.tsx

```diff
@@ -36,6 +36,11 @@ export interface FileUpload {
    */
   readonly progress: number;
 
+  /**
+   * Url file was uploaded to, used to send in mutation to Jobber Online
```

Comment:
Ah, better, but what I mean is that it's not just Jobber Online who uses this open source project. 

We need to be clear on where did this URL came from and what is it for. Looking at the code it is just a thing that came from what you passed in the `getUploadParams`, right? In theory, you would have this already without doing a change here but in a sense of ease-of-use, sure let's add it.

Also raises the question of whether this should be optional if `getUploadParams` and the `url` it returns are both required?

Reply:
I copied that from mobile code, but yes, as `getUploadParams` and `url` will always be defined so this one is never undefined and not optional

Reply:
That's fair, but, in the context of this repo, how should we write that? 🤔

Reply:
`Url file was uploaded to` maybe is enough? As this is just the base url not the full file url

Reply:
ah, that prop should've been named `baseUrl` but for the sake of consistency, let's stick with `url`.

You can also use the existing one where this came from as an inspiration

https://github.com/GetJobber/atlantis/blob/8f33bd6f75d9fb4e241e98dbe2ac46588dbc665d/packages/components/src/InputFile/InputFile.tsx#L47

> Base URL where file was sent as a POST. This is the same URL as what's returned in `getUploadParams`.